### DISCUSSION TITLE
v5.3.0-alpha.1

### DISCUSSION THREAD

**Discussion by coolsoftwaretyler** [mobxjs/mobx-state-tree#2101] [General] [answered]
This alpha release was built and shipped from the branch in https://github.com/mobxjs/mobx-state-tree/pull/2099. We did that to make sure none of the configuration changes broke our deployment processes, but also shipped some changes.

Once we get a few weeks of usage data and verify things are good, we can ship a full `5.3.0`. I'm feeling optimistic about doing that shortly.

## Breaking Changes

* No breaking changes

## Features

* Add number types to map model methods by @a-hassanzadeh-h in https://github.com/mobxjs/mobx-state-tree/pull/2079

## Fixes

* perf: use push instead of unshift by @BrianHung in https://github.com/mobxjs/mobx-state-tree/pull/2082
* use mobx type for map initial data by @BrianHung in https://github.com/mobxjs/mobx-state-tree/pull/2072
* perf: use map instead of weak map by @BrianHung in https://github.com/mobxjs/mobx-state-tree/pull/2084
* return undefined in resolveNodeByPathParts when failIfResolveFails = false by @BrianHung in https://github.com/mobxjs/mobx-state-tree/pull/2071

## Docs

* Add union tips by @chakrihacker in https://github.com/mobxjs/mobx-state-tree/pull/2078
* Clarify wording and fix old link in philosophy.md by @StefanWallin in https://github.com/mobxjs/mobx-state-tree/pull/2093
* Fix wording and discuss a bad link in philosophy.md by @StefanWallin in https://github.com/mobxjs/mobx-state-tree/pull/2094

## Development/community

* chore: add community docs by @coolsoftwaretyler in https://github.com/mobxjs/mobx-state-tree/pull/2090
* chore: remove immer open collective link by @coolsoftwaretyler in https://github.com/mobxjs/mobx-state-tree/pull/2086
* Chore/remove mst middlewares package by @coolsoftwaretyler in https://github.com/mobxjs/mobx-state-tree/pull/2091

## New Contributors
* @StefanWallin made their first contribution in https://github.com/mobxjs/mobx-state-tree/pull/2093
* @a-hassanzadeh-h made their first contribution in https://github.com/mobxjs/mobx-state-tree/pull/2079

**Full Changelog**: https://github.com/mobxjs/mobx-state-tree/compare/5.2.0...v5.3.0-alpha.1

<hr /><em>This discussion was created from the release <a href='https://github.com/mobxjs/mobx-state-tree/releases/tag/v5.3.0-alpha.1'>v5.3.0-alpha.1</a>.</em>

**Comment by cmdcolin** [comment]
this alpha passes all tests on our project :) thanks for pushing mst forward @coolsoftwaretyler it is great to see

**Reply by coolsoftwaretyler** [reply, marked as answer]
Thanks for testing and letting us know @cmdcolin! It really helps a lot just to hear from folks.

I've been running it in production for about a week now without issue as well. Planning to mark it as latest early next week unless I hear of any issues.
